code/datums/status_effects/buffs.dm

```diff
@@ -748,6 +748,10 @@
 	duration = BRAIN_DAMAGE_MOB_TIME
 	alert_type = null
 
+/datum/status_effect/expert_chef
+	duration = 20 SECONDS
```

Comment:
I understand using a status effect to avoid spamming it, but 20 seconds seems overly long for something that's purely informational.